src/rules/rule-non-nested-empty-line-before/__tests__/index.js

```diff
@@ -49,6 +49,28 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { except: ["after-single-line-comment"] } ],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "/**\n * comment\n*/\n\na {}",
+  }, {
+    code: "/**\n\t* comment\n*/\na {}",
```

Comment:
> I'm pretty sure I've got the tests accurate

Just want to clarify so I understand the change ... You're saying that the following CSS should be accepted:

```css
/**
    * comment
*/
a {}
```

Shouldn't that fail because there's no empty line after a multi-line comment?